### PR TITLE
add missing extra-dep in stack.yaml

### PR DESCRIPTION
**Pull request by KtorZ**
```console
$ stack install 
Error: While constructing the build plan, the following exceptions were encountered:

In the dependencies for cardano-api-1.13.0:
    base64 needed, but the stack configuration has no specified version  (latest matching version is 0.4.1)
needed since cardano-api is a build target.

Some different approaches to resolving this:

  * Recommended action: try adding the following to your extra-deps in /home/ktorz/Documents/IOHK/cardano-node/stack.yaml:

- base64-0.4.1@sha256:aedc26f581f81a112af588eec8cf9c2e8db219a60fafc1cb77fd73672f1f7dcb,2832

Plan construction failed.
```

### CODE REVIEW

File: cardano-cli/src/Cardano/CLI/Shelley/Run/Genesis.hs [hunk 1]
```diff
@@ -126,10 +126,10 @@ renderShelleyGenesisCmdError err =
         <> "delegate VRF key file indexes: " <> textShow vfiles
     ShelleyGenesisFilesNoIndex files ->
       "The genesis keys files are expected to have a numeric index but these do not:\n"
-        <> unlines (map Text.pack files)
+        <> Text.unlines (map Text.pack files)
     ShelleyGenesisFilesDupIndex files ->
       "The genesis keys files are expected to have a unique numeric index but these do not:\n"
-        <> unlines (map Text.pack files)
+        <> Text.unlines (map Text.pack files)
```
Comment (KtorZ) [inline cardano-cli/src/Cardano/CLI/Shelley/Run/Genesis.hs:132]:
```
cardano-cli> [19 of 30] Compiling Cardano.CLI.Shelley.Run.Genesis
cardano-cli> 
cardano-cli> /home/ktorz/Documents/IOHK/cardano-node/cardano-cli/src/Cardano/CLI/Shelley/Run/Genesis.hs:129:12: error:
cardano-cli>     • Variable not in scope: unlines :: [Text] -> Text
cardano-cli>     • Perhaps you meant one of these:
cardano-cli>         ‘BS.unlines’ (imported from Data.ByteString.Char8),
cardano-cli>         ‘List.unlines’ (imported from Data.List),
cardano-cli>         ‘Text.unlines’ (imported from Data.Text)
cardano-cli>     |
cardano-cli> 129 |         <> unlines (map Text.pack files)
cardano-cli>     |            ^^^^^^^
cardano-cli> 
cardano-cli> /home/ktorz/Documents/IOHK/cardano-node/cardano-cli/src/Cardano/CLI/Shelley/Run/Genesis.hs:132:12: error:
cardano-cli>     • Variable not in scope: unlines :: [Text] -> Text
cardano-cli>     • Perhaps you meant one of these:
cardano-cli>         ‘BS.unlines’ (imported from Data.ByteString.Char8),
cardano-cli>         ‘List.unlines’ (imported from Data.List),
cardano-cli>         ‘Text.unlines’ (imported from Data.Text)
cardano-cli>     |
cardano-cli> 132 |         <> unlines (map Text.pack files)
cardano-cli>     |            ^^^^^^^
cardano-cli> 
```

File: stack.yaml [hunk 2]
```diff
@@ -27,6 +27,7 @@ extra-deps:
   - micro-recursion-schemes-5.0.2.2
   - streaming-binary-0.3.0.1
   - canonical-json-0.6.0.0
+  - base64-0.4.1
```
Comment (CodiePP) [inline stack.yaml:30]:
```suggestion
  - base64-0.4.2
  - ghc-byteorder-4.11.0.0.10
```
I needed to insert these new extra-deps to compile it using `stack`

Comment (KtorZ) [reply stack.yaml:30]: Ah possible. I've compiled 1.13.0, master has probably already diverged.

Comment (KtorZ) [reply stack.yaml:30]: Actually no, it requires `base64-0.4.1` also on latest master.

**Pull request merged**